File: src/main/java/service/ContentService.java
package service;

import dao.ContentDao;
import model.contentModel;

import java.util.Date;

public class ContentService {

    private ContentDao contentDao;

    public ContentService() {
        contentDao = new ContentDao();
    }

    // Method to create content
    public void createContent(String title, String body, String author) {
        contentModel content = new contentModel(title, body, author, new Date());
        contentDao.saveContent(content);
    }
}

File: src/main/scala/service/contentService.scala
package service;

import com.scalatutorial.dao.ContentDao
import com.scalatutorial.model.Content
import java.util.Date

class contentService(contentDao: ContentDao) {

  // Method to create content
  def createContent(title: String, body: String, author: String): Content = {
    val content = new Content(title = title, body = body, author = author, createdAt = new Date())
    contentDao.saveContent(content)
    content
  }
}
